patches/tModLoader/Terraria/Recipe.TML.cs

```diff
@@ -460,6 +460,12 @@ public Recipe Register()
 		return this;
 	}
 
+	/// <summary>
+	/// Create the recipe but does not yet register it into the game. Call this at the very beginning when creating a new craft.
+	/// </summary>
+	/// <param name="result">What item will be given when the craft has been completed</param>
+	/// <param name="stack">The stack -> how many result items given when the craft is completed. (eg. 1 wood -> 4 wood platform)</param>
```

Comment:
The param name is actually "amount" here. It could also be worth pointing out the default of 1, so you only need to pass a value if you want something other than 1 of the item per craft.